public/config/configuration.json

```diff
@@ -6,6 +6,7 @@
   "LEGAL_API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "VUE_APP_STATUS_ROOT_API": "https://status-api-dev.pathfinder.gov.bc.ca/api/v1",
```

Comment:
Add a comment to the release ticket (/bcgov/entity#4672) to remember to update the config maps.

Reply:
Copy that, will do!

Reply:
This has been updated in Dev And Test, currently 👍